doc/nrf/protocols/matter/getting_started/memory_optimization.rst

```diff
@@ -0,0 +1,40 @@
+.. _ug_matter_device_optimizing_memory:
+
+Optimizing memory usage in Matter applications
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use different approaches to optimizing the memory usage of your Matter application, both on the |NCS| side and in the Matter SDK.
+
+Reducing memory usage on the |NCS| side
+***************************************
+
+See the :ref:`app_memory` guide for information about how to reduce memory usage for the |NCS| generally and for specific subsystems in particular, including Bluetooth® LE, Matter, and Thread.
+
+.. _ug_matter_device_optimizing_memory_logs:
+
+Cutting off log regions for Matter SDK modules
+**********************************************
+
+The Matter SDK's nRF Connect platform, included in the |NCS| as one of the submodule repositories using a `dedicated Matter fork`_, provides a custom mechanism for optimizing memory usage in a Matter application.
```

Comment:
It's actually not a custom mechanism of our platform. Logs in Matter can be configured in the same way for every platform.